### PR TITLE
fix(package.json): consistent props

### PR DESCRIPTION
**Pull request by evenstensberg**
### Describe your changes:
package.json in e2e and root metadata should be identical.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Comment by evenstensberg** [comment]
@ozyx lmk if there's anything I need to do before it is mergeable :)

**Comment by ozyx** [comment]
> @ozyx lmk if there's anything I need to do before it is mergeable :)

Thanks @evenstensberg ! Really appreciate your patience with us. I'll have a look this afternoon. Cheers

**Comment by evenstensberg** [comment]
🌟

File: e2e/package.json [hunk 1]
```diff
@@ -21,6 +21,10 @@
     "@axe-core/playwright": "4.8.5",
     "sinon": "17.0.0"
   },
-  "author": "NASA Ames Research Center",
+  "homepage": "https://github.com/nasa/openmct/tree/master/e2e",
```
Comment (ozyx) [inline e2e/package.json:24]: let's remove the `homepage` prop since this is our e2e testing framework and it will not be served

**Review by ozyx — CHANGES_REQUESTED** [review]
One small change requested

**Review by ozyx — APPROVED** [review]
LGTM, thanks!

**Pull request merged**